firefox_desktop/views/newtab_interactions.view.lkml

```diff
@@ -195,6 +195,20 @@ view: newtab_interactions {
     sql: ${tagged_follow_on_search_ad_clicks} ;;
   }
 
+  measure: sum_all_search_ad_clicks {
+    group_label: "Search"
+    label: "Total Ad Clicks"
+    type: sum
+    sql: ${tagged_search_ad_clicks} + ${follow_on_search_ad_clicks} ;;
```

Comment:
I don't think these are mutually exclusive - see above `sum_tagged_follow_on_search_ad_clicks`, it's defined AS:

```
  COUNTIF(
    is_tagged_search_ad_click
    AND is_follow_on_search_ad_click
  ) AS tagged_follow_on_search_ad_clicks,
```
Same for ad impressions

Reply:
Ah ok, so this means that follow on is a subset of tagged searches then. This is different than our usual way we define these things, let me make the appropriate adjustments.

Reply:
I've made some edits, can you take a look @ANich? 
Do we need to update the dimensions definitions in the view as well? And should we add labels to all of these metrics?

Reply:
See https://github.com/mozilla/looker-spoke-default/pull/452#issuecomment-1325227620, this should probably be updated at the source so it doesn't cause confusion elsewhere. Yeah this could definitely use some descriptions